Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -524,6 +524,11 @@ public bool OpenSolution(string solutionFolder, string encryptionKey = null)
                 BusinessFlow.SolutionVariables = solution.Variables;
                 solution.SolutionOperations.SetReportsConfigurations();
                 Solution = solution;
+                
+                string txt = File.ReadAllText(solutionFile);
```

Comment:
these 2 lines are needed?

Reply:
Calling the "NewRepositorySerializer.DeserializeFromText(txt)" function invokes the "SerializationError" function. I have place it below the code where Solution object is initialized because the SerializationError function in the HTMLReportsConfiguration, I reference this Solution object to get the currently used ExecutionLoggerConfiguration object and set the respective HTMLURL attribute.

Reply:
Will remove the 2 lines and need to check better way to handle it